spec/spec.md

```diff
@@ -23,6 +23,7 @@ This specification uses [semantic versioning](https://en.wikipedia.org/wiki/Soft
 | Version | Description                                                                                                                         |
 | ------- | ----------------------------------------------------------------------------------------------------------------------------------- |
 | 1.0.0   | Initial release version of the Mobile Wallet Adapter specification (identical to pre-release version 0.9.1)                         |
+| 2.0.0   | Initial draft of Mobile Wallet Adaper 2.0                                                                                           |
```

Comment:
NIT: we should only formally rev the version on the `main` branch when this is no longer in draft